Package.swift

```diff
@@ -0,0 +1,39 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Clickstream",
+    platforms: [.macOS(.v10_12),
+                .iOS(.v11),
+                .tvOS(.v10),
+                .watchOS(.v3)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Clickstream",
+            targets: ["Clickstream"]),
+    ],
+    dependencies: [
+        .package(name: "SwiftProtobuf", url: "https://github.com/apple/swift-protobuf.git", from: "1.6.0"),
+        .package(name: "Reachability", url: "https://github.com/ashleymills/Reachability.swift", from: "5.0.0"),
+        .package(name: "GRDB", url: "https://github.com/groue/GRDB.swift.git", from: "5.12.0"),
+        .package(name: "Starscream", url: "https://github.com/daltoniam/Starscream.git", from: "4.0.0"),
```

Comment:
In Podfile the version of Starscream is 4.0.4. Also in Package.resolved its 4.0.4

Reply:
Yes I updated it to be of version 4.0.4, similar to pod